CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 0.7.0 - Unreleased
+
+- adapt to the new Hammer API
```

Comment:
```suggestion
- adapt to the new Hammer API. Must use 7.x release at a minimum
```

Reply:
This is a dependency resolution issue, not a documentation issue. It should not be possible to use incompatible versions.

Reply:
while I agree, letting them know does not hurt. if you look at the mix file, it currently would allow the upgrade while not compatible and could confuse the user. So being explicit in the changelog provide that extra information

Reply:
The new version would depend on Hammer v7, so the upgrade wouldn't happen automatically.